data/json/recipes/recipe_weapon.json

```diff
@@ -6733,5 +6733,52 @@
     ["steel_chunk", 4],
     ["scrap", 12]]
   ]
+},
+{
+  "type" : "recipe",
+  "result": "chemical_thrower",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_RANGED",
+  "skill_used": "mechanics",
+  "skills_required": [[ "gun", 3 ], [ "cooking", 3]],
+  "difficulty": 7,
+  "time": 12000,
+  "reversible": true,
+  "autolearn": true,
+  "qualities":[
+    {"id":"SAW_M_FINE","level":1,"amount":1},
+    {"id":"SCREW_FINE","level":1,"amount":1},
+	{"id":"HAMMER","level":2,"amount":1}
+  ],
+  "tools": 
+  [
+    [
+      ["goggles_welding", -1]
+    ],
+    [
+      ["oxy_torch", 80],
+      ["welder", 400],
+      ["welder_crude", 550],
+      ["toolset", 550]
+    ],
+  ], "components": [
+    [
+      [ "pilot_light", 1 ]
+    ],
+	[
+	  [ "wrench", 1]
```

Comment:
Wrenches can be quire valuable here. Since the recipe involves welding, a pipe would be enough here.

Also, add a small or tiny metal tank to the recipe.